superset/views/core.py

```diff
@@ -1530,16 +1536,18 @@ def checkbox(self, model_view, id_, attr, value):
 
     @api
     @has_access_api
-    @expose('/schemas/<db_id>/')
-    def schemas(self, db_id):
+    @expose('/schemas/<db_id>/<force_refresh>/')
+    def schemas(self, db_id, force_refresh):
```

Comment:
It would be good to preserve the old behavior, just in case something is using the API. I think you can do here:

```python
@expose('/schemas/<db_id>/<force_refresh>/')
@expose('/schemas/<db_id>/')
def schemas(self, db_id, force_refresh=True):
```

And it would handle both cases.